src/models/nebula36-1/assembly.md

```diff
@@ -0,0 +1,399 @@
+
+![CPU shroud back fan bracket](./img/cpu-shroud-fan-bracket-back.webp)
+
+3. Unscrew the four fan screws holding the fan onto the bracket.
+
+![CPU shroud rear exhaust fan](./img/cpu-shroud-rear-fan-screws.webp)
+
+4. When reinstalling the fan, the stationary cover should face the back of the chassis, and the wire should face the upper interior corner of the CPU shroud. The thinnest side of the bracket should face directly into the shroud.
+    - The wire can be run along the top of the fan until it meets the velcro strap.
+
+## Installing the GPU:
+
+nebula36 supports GPUs up to 318.80mm long. For optimal cooling and ease of installation, GPUs longer than 308.80mm are not recommended.
+
+These instructions also apply to other PCI Express cards, such as add-in sound cards or networking cards.
+
+- **Tools required:** Cross-head (Phillips) screwdriver  
+
+### Steps to install the GPU:
+
+1. If not already installed, [install the motherboard](#installing-the-motherboard).
+    - If the motherboard is already installed and the system is assembled, then [remove the top case](#removing-the-top-case) and [remove the GPU brace](#removing-the-gpu-brace).
+2. Unscrew the two screws holding the PCIe bracket onto the chassis, then remove the bracket.
+
+![PCIe Bracket](./img/pcie-bracket.webp)
+
+3. Remove a PCIe slot cover for each slot the new GPU will take up.
+    - Additional slot covers may fall off after removing the PCIe brakcet; hold them up when replacing the bracket.
+4. Install the GPU into the motherboard.
+5. Replace the PCIe bracket, which will hold the end of the GPU in place.
+
+## Installing the SATA backplane:
+
+nebula36 is designed to work with an optional SATA backplane to allow for easy hot-swapping of 2.5" storage drives. The SATA backplane can be seen below.
+
+![2.5" SATA backplane](./img/sata-backplane.webp)
+
+- **Tools required:** Cross-head (Phillips) screwdriver  
+
+### Steps to install the SATA backplane:
+
+1. [Remove the top case](#removing-the-top-case).
+2. Unscrew and remove the 2.5" drive cage's cover.
+
+![2.5" SATA drive cage cover](./img/25-drive-cover.webp)
+
+3. Unscrew and remove the 2.5" drive cage.
+
+![2.5" SATA drive cage](./img/25-drive-cage-screws.webp)
+
+4. If the SATA backplane's screw holes are covered by a translucent film, remove the film.
+5. Place the SATA backplane into the drive cage.
+    - The backplane should be oriented so the power plug will be at the top of the board after installing the drive cage back into the chassis.
+6. Screw the backplane into the drive cage from the outside.
+
+![2.5" SATA backplane screws](./img/sata-backplane-screws.webp)
+
+7. Screw the drive cage back onto the chassis.
+8. Connect the `POWER0` header on the back of the SATA backplane to the power supply, and the `DATA0` and `DATA1` ports to two of the motherboard's SATA ports.
```

Comment:
Berg connector should be specified here too (if you didn't add it already)

Reply:
Yep, already addressed

Reply:
Already done.